### PR TITLE
[Enhancement] Change readme style and Update metafiles.

### PR DESCRIPTION
**Pull request by sennnnn**
- [x] Improve readme stryle;
- [x] Modify md2yml hooks;
- [x] Update metafiles;

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ Supported methods:
 - [x] [PointRend (CVPR'2020)](configs/point_rend)
 - [x] [CGNet (TIP'2020)](configs/cgnet)
 - [x] [SETR (CVPR'2021)](configs/setr)
+- [x] [DPT (ArXiv'2021)](configs/dpt)
```
Comment (Junjun2016) [inline README.md:97]: Also, add to README_zh-CN.md.

**Comment by Junjun2016** [comment]
Hi @sennnnn 
Please resolve the conflicts.

File: configs/ann/ann.yml [hunk 2]
```diff
@@ -1,296 +1,306 @@
...
+      backend: PyTorch
+      batch size: 1
       mode: FP32
       resolution: (512,1024)
-      value: 269.54
-    lr schd: 40000
     memory (GB): 6.0
-  Name: ann_r50-d8_512x1024_40k_cityscapes
   Results:
+    Task: Semantic Segmentation
```
Comment (Junjun2016) [inline configs/ann/ann.yml:34]:
```suggestion
    -Task: Semantic Segmentation
```

Refer to https://github.com/open-mmlab/mmdetection/blob/master/configs/atss/metafile.yml#L36.

File: configs/apcnet/apcnet.yml [hunk 3]
```diff
@@ -1,223 +1,233 @@
...
-  Name: apcnet
+  Paper:
+    URL: https://openaccess.thecvf.com/content_CVPR_2019/html/He_Adaptive_Pyramid_Context_Network_for_Semantic_Segmentation_CVPR_2019_paper.html
+    Title: Adaptive Pyramid Context Network for Semantic Segmentation
+  README: configs/apcnet/README.md
+  Code:
+    URL: https://github.com/open-mmlab/mmsegmentation/blob/v0.17.0/mmseg/models/decode_heads/apc_head.py#L111
+    Version: v0.17.0
+  Converted From:
+    Weights: ''
```
Comment (Junjun2016) [inline configs/apcnet/apcnet.yml:15]: pop the `key` if the length of the corresponding value is 0.

**Review by Junjun2016 — APPROVED** [review]
LGTM

**Review by Junjun2016 — APPROVED** [review]

**Review by xvjiarui — APPROVED** [review]

**Pull request merged**